### DISCUSSION TITLE
Add ability to type in the multiplayer room chat while map is playing

### DISCUSSION THREAD

**Discussion by Szago** [ppy/osu#14340] [Ideas] [answered]
One of the multiplayer aspects that I'm missing in lazer right now is being able to chat with people in the lobby while loading or during breaks, also in the future - things like notifying referees in the match that something happened and map needs to be aborted etc. There should be some kind of access to the multiplayer chat while mid map in the F8 menu.

**Comment by peppy** [comment, marked as answer]
It should already be there, can you not see it?

**Reply by Szago** [reply]
I'm able to open F8 menu during break in the map, but there is no access to the room specific chatroom, unless I'm missing something?

**Reply by frenzibyte** [reply]
There's a chat specific to the room you're currently in, but either way there is an issue open proposing similar to what you're proposing https://github.com/ppy/osu/issues/12402, so this would be a duplicate of it.
